extensions/ql-vscode/src/extension.ts

```diff
@@ -493,6 +494,32 @@ async function activateWithInstalledDistribution(
     }
   }
 
+  const tmpDir = tmp.dirSync({ prefix: 'qhelp_', keep: false, unsafeCleanup: true });
```

Comment:
Also I think this pattern does the right thing on Windows (since we use it elsewhere in the extension), but I would not object to a second look from @shati-patel.

Reply:
> Also I think this pattern does the right thing on Windows (since we use it elsewhere in the extension), but I would not object to a second look from @shati-patel.

Looks good to me too! 😊